js/interface.js

```diff
@@ -291,8 +291,73 @@
         currentProvider = null;
         Fliplet.Studio.emit('reload-page-preview');
       });
+    })
+    .on('click', '[data-select-icon]', function() {
+      var itemId = $(this).parents('.panel').data('id');
+      var currentItem = _.find(currentMenuItems, function(item) {
+        return item.id === itemId;
+      });
+      initIconProvider(currentItem);
+    })
+    .on('click', '.remove-icon', function() {
+      var itemId = $(this).parents('.panel').data('id');
+      var currentItem = _.find(currentMenuItems, function(item) {
+        return item.id === itemId;
+      });
+      var iconBak = currentItem.data.icon;
+      currentItem.data.icon = undefined;
+      $(this).parents('.icon-selection').removeClass('icon-selected');
```

Comment:
you could have saved `$(this).parents('.icon-selection')` into a variable instead of doing 3 DOM calls :)